src/Combinatorics/Graphs/functions.jl

```diff
@@ -35,6 +35,68 @@ function Graph{T}(nverts::Int64) where {T <: Union{Directed, Undirected}}
     return Graph{T}(pmg)
 end
 
+@doc raw"""
+    graph_from_adjacency_matrix(::Type{T}, G) where {T <:Union{Directed, Undirected}}
+
+Return the graph with adjacency matrix `G`.
+
+This means that the nodes ``i > j`` are connected by an edge
+if and only if ``G_{i,j}`` is one.
+
+# Examples
+```jldoctest
+julia> G = ZZ[0 0; 1 0]
+[0   0]
+[1   0]
+
+julia> graph_from_adjacency_matrix(Directed, G)
+Graph{Directed}(pm::graph::Graph<pm::graph::Directed>
+{}
+{0}
+)
+
+julia> graph_from_adjacency_matrix(Undirected, G)
+Graph{Undirected}(pm::graph::Graph<pm::graph::Undirected>
+{1}
+{0}
+)
```

Comment:
Yes. Once the show functions of the graphs improve, this doctests will be more helpful. 
For now it is just intended to show how to use the constructor and so that the user has something to copy paste.